src/Akeneo/Connectivity/Connection/back/Infrastructure/Persistence/Elasticsearch/Query/SearchEventSubscriptionDebugLogsQuery.php

```diff
@@ -128,33 +172,22 @@ public function execute(string $connectionCode, ?string $encryptedSearchAfter =
         ];
 
         if ($filters['levels']) {
-            $query['query']['bool']['must'] = ['terms' => ['level' => $filters['levels']]];
+            $query['query']['bool']['must']['terms']['level'] = $filters['levels'];
         }
 
-        if (null !== $parameters['search_after']) {
-            $query['search_after'] = $parameters['search_after'];
+        if ($filters['timestamp_from']) {
```

Comment:
```suggestion
        if (null !== $filters['timestamp_from']) {
```
Because `0 == false` (event if a timestamp of 0 should not happen)